com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Messages.java

```diff
@@ -519,7 +519,7 @@
     public static final String GETTING_APPLICATIONS_BY_METADATA_LABEL_SELECTOR_0 = "Getting applications by metadata label selector \"{0}\"...";
     public static final String UPDATING_METADATA_OF_APPLICATION_0_TO_1 = "Updating metadata of application \"{0}\" to: {1}";
     public static final String GETTING_SERVICE_INSTANCES_BY_METADATA_LABEL_SELECTOR_0 = "Getting service instances by metadata label selector \"{0}\"...";
-    public static final String UPDATING_METADATA_OF_SERVICE_INSTANCE_0_TO_1 = "Updating metadata of service instance \"{0}\": {1}";
+    public static final String UPDATING_METADATA_OF_SERVICE_INSTANCE_0_TO_1 = "Updating metadata of service instance \"{0}\" to: {1}";

```

Comment:
Will we see something sensitive in metadata?

Reply:
We might if it's not blacklisted in `SecureSerializerConfiguration`, but at this point I prefer leaving the metadata in the logs, since it might help with the debugging of some issues.